### PR TITLE
fix(firefox-arm64-bin): force creation of /lib/ folder

### PR DESCRIPTION
**Pull request by oklopfer**

### CODE REVIEW

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript [hunk 1]
```diff
@@ -7,10 +7,15 @@ description="Free and open-source web browser developed by the Mozilla Foundatio
...
+build() {
+    sudo mkdir -p "${pkgdir}/usr/lib/"
+    sudo mkdir -p "${pkgdir}/usr/lib/firefox"
+}
+
 install() {
     cd ..
     sudo install -Dm755 "usr/bin/firefox" -t "${pkgdir}/usr/bin/"
-    sudo cp -r "usr/lib/firefox/" "${pkgdir}/usr/lib/firefox"
+    sudo cp -r "usr/lib/firefox/*" "${pkgdir}/usr/lib/firefox/"
```
Comment (Henryws) [inline packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:18]:
```suggestion
    sudo cp -r "usr/lib/firefox/"* "${pkgdir}/usr/lib/firefox/"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/firefox-arm64-bin/firefox-arm64-bin.pacscript (hunk 1 above)
Comment (oklopfer) [reply packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:18]: ??
I am copying all of the files within the firefox folder to the new firefox folder; the * should be inside the quotes

Comment (Henryws) [reply packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:18]: Did you test it?

Comment (oklopfer) [reply packages/firefox-arm64-bin/firefox-arm64-bin.pacscript:18]: affirmative, the issue was with me not setting up the dir in pacscript

**Review by Henryws — APPROVED** [review]

**Pull request merged**